flag_engine/identities/models.py

```diff
@@ -28,20 +47,23 @@ class IdentityModel:
     identity_features: typing.List[FeatureStateModel] = field(default_factory=list)
     identity_traits: typing.List[TraitModel] = field(default_factory=list)
 
+    def get_identity_traits_as_dict(self) -> typing.Dict[str, TraitModel]:
+        return {trait.trait_key: trait for trait in self.identity_traits}
+
     def get_all_feature_states(
-        self, environment: EnvironmentModel
+        self, environment: EnvironmentModel, traits: typing.List[TraitModel] = None
     ) -> typing.List[FeatureStateModel]:
         all_feature_states = {fs.feature: fs for fs in environment.feature_states}
+        with override_identity_traits(self, traits or self.identity_traits) as _:
```

Comment:
I don't think we need the `as _` bit? You don't _need_ to capture the output from a context manager I don't think?